README.md

```diff
@@ -77,10 +75,15 @@ Create VSCode Project | As well as the CMake files, also create the appropriate
 Debugger | Use the specified debugger in the IDE
 
 
+## Optional: Installation using Python Pip (Linux)
 
+This application can **optionally** be installed to your system using standard Python installation tools. If you would like to do so, follow the instructions below. If you do not wish to install the application, you can run it from the the pico_project directory.
 
+Clone the repository with: `$ git clone https://github.com/raspberrypi/pico-project-generator` (assuming you are using HTTP to clone repositories). Change to the repository's directory with: `$ cd pico-project-generator`.
```

Comment:
I don't think this "(assuming you are using HTTP to clone repositories)" bit is necessary?